gemd/entity/value/tests/test_inchi.py

```diff
@@ -14,13 +14,16 @@ def test_json():
 
 
 def test_inchi_setter():
-    """Check that we can set the formula."""
+    """Check that we can set the inchi."""
     test_struct = "InChI=1/C8H8O3/c1-11-8-4-6(5-9)2-3-7(8)10/h2-5,10H,1H3"
     test_inchi = InChI()
     assert test_inchi.inchi is None
     test_inchi.inchi = test_struct
     assert test_inchi.inchi == test_struct
 
+    assert InChI("inchi=1/C8H8O3").inchi == "InChI=1/C8H8O3"
+    assert InChI("C8H8O3").inchi == "InChI=1S/C8H8O3"
```

Comment:
```suggestion
    assert InChI("C8H8O3").inchi == "InChI=1S/C8H8O3"
    assert InChI("1/C8H8O3").inchi == "InChI=1/C8H8O3"
    assert InChI("1s/C8H8O3").inchi == "InChI=1s/C8H8O3"
```

Reply:
Good add, though the standard expects a capital S.